app/__init__.py

```diff
@@ -210,6 +210,12 @@ def load_user(user_id: str) -> Optional["User"]:
     app.jinja_env.add_extension("jinja2.ext.i18n")
     app.jinja_env.add_extension("jinja2.ext.do")
 
+    def get_google_tag_manager_id() -> str:
+        return str(current_app.config["GOOGLE_TAG_MANAGER_ID"])
+
+    def get_current_env_name() -> str:
+        return str(current_app.config["FLASK_ENV"].value)
+
```

Comment:
nit: these could be `@property`'s as callable things feel unnecessary, but it's the tiniest of optional comments and not worth rebasing for probably